target-design/switch/flit.h

```diff
@@ -49,7 +49,7 @@ void write_valid_flit(uint8_t * send_buf, int tokenid) {
     *lrv |= (1L << bitoffset);
 }
 
-int write_last_flit(uint8_t * send_buf, int tokenid, int is_last) {
+void write_last_flit(uint8_t * send_buf, int tokenid, int is_last) {
```

Comment:
@sagark , this is what was causing the switch to segfault.  Don't ask me how this causes it, I just know that when I fixed the return type of this one function, it stops happening (at least the handful of times I've run it).  I have poweroff running in a loop for awhile on my EC2.  After it does it without the switch dying a few more times, I'll run iperf3 like we talked about.

Reply:
oh wow, thanks for figuring this out!

Reply:
oh wow, thanks for figuring this out!